webapp/views/partials/includes/banner.jade

```diff
@@ -1,3 +1,4 @@
+-var menu = {'Home': 'home', 'Users': 'users'};
```

Comment:
This part "defines" the state, its just and array of variables in the jade for the statefulness